include/deal.II/fe/fe_values.h

```diff
@@ -313,6 +364,13 @@ namespace FEValuesViews
                                  std::vector<typename ProductType<gradient_type,typename InputVector::value_type>::type> &gradients) const;
 
     /**
+     * Same as above, but using a vector local degree-of-freedom values.
```

Comment:
same changes as above.

people don't usually scroll up to find a code snippet for some other function. Maybe copy the code snippet everywhere where it's relevant?